konfigure-android/src/main/java/nz/co/trademe/konfigure/android/ui/view/ConfigPresenter.kt

```diff
@@ -34,7 +34,12 @@ internal class ConfigPresenter(
     /**
      * Property acting as a search trigger for triggering async searching
      */
-    private val searchTrigger = MutableSharedFlow<Unit>(replay = 1, onBufferOverflow = BufferOverflow.DROP_LATEST)
+    private val searchTrigger = MutableSharedFlow<Unit>(
+        replay = 1,
+        onBufferOverflow = BufferOverflow.DROP_LATEST
+    ).apply {
+        tryEmit(Unit)
```

Comment:
Instead of doing this here when the class/field is instantiated, I think it might better be controlled by the caller? e.g. `ConfigView` can kick off a search when  `onViewCreated` or some other time.

Reply:
it happens that we use filter, emitting to `searchTrigger` happens earlier than that window attached (so would be potentially before we trigger that by caller). So I'm not sure what benefit it gives us in the case. Unless you see something I'm missing here
Also we already have `changeNotifier` that's self initializing with null first in the ConfigPresenter, so thought using same pattern would be more readable